scripts/generateDocs.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs')
+const path = require('path')
+
+const README_DIR = './src/stories/README';
+const REPLACE_TARGET = '{% COMPONENT_DOCS %}'
+
+const fileNames = fs.readdirSync('./src/stories/README')
+
+const docs = []
+
+for (const name of fileNames) {
+  const file = fs.readFileSync(path.join(README_DIR, name), 'utf8')
+  // Reduce size of markdown headers to better fit the main README format
+  docs.push(file.replace(/(^|\n)#/g, '$&##'))
+}
+
+const content = fileNames.map((file) => {
+  const name = file.replace('.md', '')
```

Comment:
We might want to run prettier against this (even if it's just once and not as part of our build process) since in some places we use `;` while we don't in other.

Reply:
good idea